### DISCUSSION TITLE
Info Zigbee SonOff and SLZB-06p7 network extension

### DISCUSSION THREAD

**Discussion by saviothecnic** [Koenkk/zigbee2mqtt#22796] [General]
Hi, I've been using zigbee2mqtt for a while
with a Son-Off Zigbee Bridge Pro ZBBridge-P Smart but needing to extend the network, I bought a SMLIGHT SLZB-06p7 POE LAN adapter with original firmware, connected it and modified the configuration files but it doesn't seem to work, where am I wrong?

Thank you
![image](https://github.com/Koenkk/zigbee2mqtt/assets/6697807/4681f678-52ec-43d7-89b7-ec7189c0839f)

**Comment by francisp2** [comment]
Where did you find secondary_serial in the docs ? I'm pretty sure it is not there.

**Reply by saviothecnic** [reply]
Yes, I think I took it as an example 
which comes out of the device's web page 
But I've done a lot of testing so I'm not sure
where did I find this info but if you can advise me
How could I fix it, I'll be happy to try

**Reply by francisp2** [reply]
You need a second Zigbee2MQTT installation
- be sure to use a different base_topic
- and a different panid
-

**Reply by saviothecnic** [reply]
my idea was to have a single extended ZigBee network 
Isn't it possible to do this?

**Reply by francisp2** [reply]
You can't transport Zigbee over Ethernet,
